### ISSUE TITLE
Sonic Boom bug in street scene

### ISSUE THREAD

**Issue by tcm390** [webaverse/app#2905]
In shadertoy.js if we enable 'SAMPLE_ALPHA_TO_COVERAGE' (`context.enable(context.SAMPLE_ALPHA_TO_COVERAGE);`) the render of sonic-boom would be wrong:

https://user-images.githubusercontent.com/60634884/165412886-01cc98ea-aa4e-400b-9db5-8eb7d3d0403f.mp4

This is the result if I comment out `context.enable(context.SAMPLE_ALPHA_TO_COVERAGE);` :

https://user-images.githubusercontent.com/60634884/165413011-b1387a1a-7073-4acd-9761-84989a8dfb5f.mp4

**Comment by tcm390** [comment]
Is it okay to disable SAMPLE_ALPHA_TO_COVERAGE?

**Comment by avaer** [comment]
> Is it okay to disable SAMPLE_ALPHA_TO_COVERAGE?

No, that would break a lot of shaders. I think perhaps the sonic boom should be updated to be compatible with alpha to coverage.
